index.html

```diff
@@ -0,0 +1,137 @@
+							</p>
+							<div class="links-container">
+								<a href="github-link.org">Github</a>
+								<a href="live-linke.org">Live</a>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class = "project">
+					<h3>Project Title</h3>
+					<div class="project-content">
+						<img src="project-image.png" alt="A nice image for the first project">
+						<div class="project-words">
+							<p>This is the first thing I would like to say about the project. This is what it was, what it did, and also this is how it did it. I like this particular thing about it, for this reason that I am saying now. It was interesting because of these reasons mostly.
+							</p>
+							<div class="links-container">
+								<a href="github-link.org">Github</a>
+								<a href="live-linke.org">Live</a>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class = "project">
+					<h3>Project Title</h3>
+					<div class="project-content">
+						<img src="project-image.png" alt="A nice image for the first project">
+						<div class="project-words">
+							<p>This is the first thing I would like to say about the project. This is what it was, what it did, and also this is how it did it. I like this particular thing about it, for this reason that I am saying now. It was interesting because of these reasons mostly.
+							</p>
+							<div class="links-container">
+								<a href="github-link.org">Github</a>
+								<a href="live-linke.org">Live</a>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class = "project">
+					<h3>Project Title</h3>
+					<div class="project-content">
+						<img src="project-image.png" alt="A nice image for the first project">
+						<div class="project-words">
+							<p>This is the first thing I would like to say about the project. This is what it was, what it did, and also this is how it did it. I like this particular thing about it, for this reason that I am saying now. It was interesting because of these reasons mostly.
+							</p>
+							<div class="links-container">
+								<a href="github-link.org">Github</a>
+								<a href="live-linke.org">Live</a>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="contact" id="contact">
+			<h2>Contact</h2>
+			<p> The best way to get in touch with me is via email, but if you prefer social media I have included these too.</p>
+			<div class="contact-container">
+				<a href="mailto:mrrobinhodge@gmail.com"><img src="emailpic.png" alt="An email thumbnail"></a>
+				<a href="https://www.linkedin.com/in/robin-hodge/"><img src="linkedinpic.png" alt="A linkedin thumbnail"></a>
+				<a href="https://github.com/horsenelly"><img src="githubpic.png" alt="A github thumbnail"></a>
+			</div>
+		</div>
+		<footer>
+			<a href="#nav">Back to top</a>
```

Comment:
nice 'back to top' functionality - personally I'd use a different name but not sure what the standard name would be (if there is one)